src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrencyGroupSizes.cs

```diff
@@ -14,9 +14,8 @@ public static IEnumerable<object[]> CurrencyGroupSizes_TestData()
             yield return new object[] { NumberFormatInfo.InvariantInfo, new int[] { 3 } };
             yield return new object[] { new CultureInfo("en-US").NumberFormat, new int[] { 3 } };
 
-            // TODO: when dotnet/corefx#2103 is addressed, we should also check fr-FR
             if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610
-                && !PlatformDetection.IsWindows7 && !PlatformDetection.IsWindows8x && !PlatformDetection.IsFedora24)
+                && !PlatformDetection.IsWindows7 && !PlatformDetection.IsWindows8x && !PlatformDetection.IsFedora24 && !PlatformDetection.IsFedora25)
```

Comment:
please replace 
```
!PlatformDetection.IsFedora24 && !PlatformDetection.IsFedora25
```

with PlatformDetection.IsFedora23 this will have us not to worry updating the tests when having version 26 and up